### DISCUSSION TITLE
Is skipping sesman now an intended feature?

### DISCUSSION THREAD

**Discussion by metcalsr28** [FreeRDP/FreeRDP#10021] [Q&A] [answered]
Until I updated today, I was able to log into my devices via a nice, simple zsh function which stored my username, the intended resolution, and the address of the machine. The procedure for launching xfreerdp3 seem to have changed such that it will prompt you in the console for your password if you don't provide it. I don't like the idea of storing the passwords to my devices in plain-text, so now I'm forced to enter the command manually or else hunt for it in my history. What exactly was wrong with simply causing a lack of password to enter you into sesman like before?

**Comment by akallabeth** [comment, marked as answer]
no, just the syntax changed slightly to distinguish between `we want no credentials`  and we just did not supply some.
add `/u: /p:` (empty credentials) and you will get the logon manager (if you have `TLS` or `RDP` security, will just fail with `NLA` of course)

**Reply by metcalsr28** [reply]
Thanks!
